packages/terra-slide-panel-manager/src/SlidePanelManager.jsx

```diff
@@ -77,6 +77,8 @@ class SlidePanelManager extends React.Component {
       panelSize = disclosureSizeToPanelSize[manager.disclosure.size];
     }
 
+    const panelBehavior = manager.disclosure.typeConfig && manager.disclosure.typeConfig.panelBehavior ? manager.disclosure.typeConfig.panelBehavior : customProps.panelBehavior;
```

Comment:
can we update this optional chaining? much cleaner :D 
 ```suggestion
     const panelBehavior = manager?.disclosure?.typeConfig?.panelBehavior ? manager.disclosure.typeConfig.panelBehavior : customProps.panelBehavior;
```

Reply:
Updated.

Reply:
@yuderekyu  and @emilyrohrbough The linter doesn't like the optional chaining syntax for some reason.

Reply:
You'll have to add the babel-parser to the eslint configuration to use optional chaining. Here's an example:

https://github.com/cerner/terra-application/blob/main/package.json#L54

Reply:
Added here f873b400d6403da9120bf3cb8de08c25f6e0fe6e